app/app/urls.py

```diff
@@ -160,10 +160,10 @@
     # images
     re_path(r'^funding/embed/?', dashboard.embed.embed, name='embed'),
     re_path(r'^funding/avatar/?', avatar.views.handle_avatar, name='avatar'),
-    re_path(r'^static/avatar/(.*)/(.*)?', avatar.views.handle_avatar, name='org_avatar'),
-    re_path(r'^static/viz/graph/(.*)?$', dataviz.d3_views.viz_graph, name='viz_graph'),
-    re_path(r'^static/viz/sscatterplot/(.*)?$', dataviz.d3_views.viz_scatterplot_stripped, name='viz_sscatterplot'),
```

Comment:
same here... these requests are pretty performance intensive.. thats why they were cached.

they're not really dynamic to the extent that we need to worry about them busting the cache either

Reply:
@owocki I renamed them to `dynamic` because they aren't truly static assets or endpoints.  They're still generated dynamically on some interval.  The CDN entries were updated before this went live to accommodate the change.  We shouldn't experience any performance hit beyond the initial recaching on the new path.

Reply:
word; i see the dynamic endpoint. makes sense